doc/xplat_cli_migration.md

```diff
@@ -42,7 +42,7 @@ using NPM, upgrade with `npm upgrade -g azure-cli`.  If you used an installer,
 we recommend downloading the latest installer to upgrade.
 
 To install the Azure CLI 2.0, follow the steps for your preferred platform or
-environment on our [Installation Guide](https://github.com/Azure/azure-cli/blob/master/README.rst#installation).
+environment on our [Installation Guide](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest).
```

Comment:
Please remove `?view=azure-cli-latest` from the URL.
Doc team would rather the server redirect to the appropriate view.